ibis/backends/dask/udf.py

```diff
@@ -0,0 +1,161 @@
+import itertools
+
+import dask.dataframe as dd
+import dask.dataframe.groupby as ddgb
+import dask.delayed
+import pandas
+
+import ibis.client
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+import ibis.udf.vectorized
+from ibis.backends.pandas.udf import nullable  # noqa
+
+from .dispatch import execute_node, pre_execute
+from .execution.util import assert_identical_grouping_keys, make_selected_obj
+
+
+@pre_execute.register(ops.ElementWiseVectorizedUDF)
+@pre_execute.register(ops.ElementWiseVectorizedUDF, ibis.client.Client)
+def pre_execute_elementwise_udf(op, *clients, scope=None, **kwargs):
+    """Register execution rules for elementwise UDFs.
+    """
+    input_type = op.input_type
+
+    # definitions
+
+    # Define an execution rule for elementwise operations on a
+    # grouped Series
+    nargs = len(input_type)
+
+    @execute_node.register(
+        ops.ElementWiseVectorizedUDF,
+        *(itertools.repeat(ddgb.SeriesGroupBy, nargs)),
+    )
+    def execute_udf_node_groupby(op, *args, **kwargs):
+        func = op.func
+
+        # all grouping keys must be identical
+        assert_identical_grouping_keys(*args)
```

Comment:
i c, you are using this here, ok then (assume you are testing a good error message)